src/components/organisms/sidebar/userMenu.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+
+import Avatar from "react-avatar";
+import { useTranslation } from "react-i18next";
+
+import { useUserStore } from "@src/store";
+
+import { Button } from "@components/atoms";
+
+import { GearIcon, PlusIcon, ReceiptIcon, UserIcon } from "@assets/image/icons";
+import { LogoutIcon } from "@assets/image/icons/sidebar";
+
+export const UserMenu = () => {
+	const { t } = useTranslation("sidebar", { keyPrefix: "menu" });
+	const { logoutFunction, user } = useUserStore();
+
+	// Mock data for organizations - replace with actual data
```

Comment:
Until we won't have the data I think we can leave it like this, if we will need to demonstrate it, we will need some data, at this moment it will be our fake.